apps/angular/crud/src/app/state/actions/todo.actions.ts

```diff
@@ -0,0 +1,43 @@
+import { createAction, props } from '@ngrx/store';
+import { Todo } from '../../model/todo.interface';
+
+export enum TodoActionName {
+  callTodoList = '[Todo] call todo list',
+  loadTodoList = '[Todo] load todo list',
+
+  callUpdateTodo = '[Todo] call update todo',
+  updateTodoSuccess = '[Todo] update todo success',
+
+  callDeleteTodo = '[Todo] call delete todo',
+  deleteTodoSuccess = '[Todo] delete todo success',
+
+  todoStatus = '[Todo] todo status',
+}
```

Comment:
This is the old way of doing it. You don't need an enum which makes it more complicated. 

Also you should look at `actionGroup` which simplified the process